package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -973,6 +973,21 @@ dnsmasq_start()
 	append_bool "$cfg" rapidcommit "--dhcp-rapid-commit"
 	append_bool "$cfg" scriptarp "--script-arp"
 
+	config_get add_mac "$cfg" add_mac
+	if [ "$add_mac" = "enabled" ] || [ "$add_mac" = "1" ]; then
+		xappend "--add-mac"
+	elif [ -n "$add_mac" ]; then
+		xappend "--add-mac=$add_mac"
+	fi
+	append_bool "$cfg" strip_mac "--strip-mac"
+	config_get add_subnet "$cfg" add_subnet
+	if [ "$add_subnet" = "0,0" ]; then
```

Comment:
```
--add-subnet[[=[<IPv4 address>/]<IPv4 prefix
length>][,[<IPv6 address>/]<IPv6 prefix length>]]
       Add  a subnet address to the DNS queries which
       are forwarded upstream. If an address is spec‐
       ified in the flag, it will be used, otherwise,
       the address of the requestor will be used. The
       amount of the address forwarded depends on the
       prefix length parameter:  32  (128  for  IPv6)
       forwards the whole address, zero forwards none
       of  it  but still marks the request so that no
       upstream nameserver will  add  client  address
       information  either.  The  default is zero for
       both IPv4 and IPv6. [omitted the rest]
```

I'm not sure if I'm understanding the man page entry right but it seems that the implied default is `0,0`, so I think this could just be shortened to:

```sh
config_get add_subnet "$cfg" add_subnet
[ -n "$add_subnet" ] && xappend "--add-subnet=$add_subnet"
```

Reply:
You're right. Will change that.